index.js

```diff
@@ -271,30 +280,30 @@ function getMIMEFromPath(path, cb) {
     });
 }
 
-//=> {ext: 'png', mime: 'image/png'}
+// => {ext: 'png', mime: 'image/png'}
 
 // gets image data from a GIF buffer
-function getBitmapFromGIF(data){
-  var gifObj = new GIF.GifReader(data);
-  var gifData = new Buffer(gifObj.width * gifObj.height * 4);
-
-  gifObj.decodeAndBlitFrameRGBA(0, gifData);
-  return {
-    data: gifData,
-    width: gifObj.width,
-    height: gifObj.height
-  };
+function getBitmapFromGIF (data) {
+    var gifObj = new GIF.GifReader(data);
+    var gifData = new Buffer(gifObj.width * gifObj.height * 4);
+
+    gifObj.decodeAndBlitFrameRGBA(0, gifData);
+    return {
+        data: gifData,
+        width: gifObj.width,
+        height: gifObj.height
+    };
 }
 
 // parses a bitmap from the constructor to the JIMP bitmap property
-function parseBitmap(data, mime, cb) {
+function parseBitmap (data, mime, cb) {
```

Comment:
I also checked and this function is not being replaced with anything else, so what's the purpose of that options object?
Can I remove it?